app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include RenderConditionConcern
+
+    check_or_render_not_found -> { enabled? }
+
+    VERIFICATION_OPTIONS = {
+      ipp: 'ipp',
+      remote: 'remote',
+    }
+
+    def show
```

Comment:
We should also add analytics for this new page for show and update ([example](https://github.com/18F/identity-idp/blob/a3e45a7022958a110cd0900168dc79a1cdea2ee3/app/controllers/idv/agreement_controller.rb#L11))